tests/Aspire.Azure.AI.OpenAI.Tests/ConformanceTests.cs

```diff
@@ -0,0 +1,108 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Aspire.Components.ConformanceTests;
+using Azure.AI.OpenAI;
+using Azure.Identity;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+
+namespace Aspire.Azure.AI.OpenAI.Tests;
+
+public class ConformanceTests : ConformanceTests<OpenAIClient, AzureOpenAISettings>
+{
+    // Fake connection string for cases when credentials are unavailable and need to switch to raw connection string
+    protected const string Endpoint = "https://aspireopenaitests.openai.azure.com/";
```

Comment:
Have you run the tests against a live openAI endpoint and ensured they still pass when you can connect to a server?

Reply:
Tested with real connection endpoint and it still works.